src/sonic-bgpcfgd/bgpcfgd/managers_bgp.py

```diff
@@ -170,6 +172,10 @@ def add_peer(self, vrf, nbr, data):
         #
         if "local_addr" not in data:
             log_warn("Peer %s. Missing attribute 'local_addr'" % nbr)
+        elif not self.require_intf_in_config_db:
```

Comment:
When the inband interfaces have been added to CONFIG_DB, we may be able to add them with another InterfaceMgr instead of doing this.